ocs_ci/ocs/ui/storageclass.py

```diff
@@ -83,11 +93,20 @@ def delete_rbd_storage_class(self, sc_name):
         self.navigate_overview_page()
         self.navigate_storageclasses_page()
         self.page_has_loaded()
+        time.sleep(5)
```

Comment:
Please remove this and use the explicit wait functions like `wait_until_expected_text_is_found` if needed.